wordpress-mobile/WordPress-iOS
# Stats Insights Latest Post Summary: abbreviate high totals

Fixes #10375 

To test:

Suggested hack to show high values: 
In `LatestPostSummaryCell:configure`, right before `self.summaryData = summaryData`, hard code the values, like so:
```
summaryData.viewsValue = 1000
summaryData.likesValue = 1234324
summaryData.commentsValue = 987654
```

- On a site where the latest post has numbers over 999 (or do the hack), go to Stats > Insights.
- Verify the numbers are abbreviated:

![some_big_numbers](https://user-images.githubusercontent.com/1816888/48171287-864f3d80-e2b8-11e8-9b8a-748dce6757a7.png)


NOTE: There are constraint warnings introduced in a previous PR. They'll be fixed with some future black magic.
